### DISCUSSION TITLE
Статусы live-трансляций

### DISCUSSION THREAD

**Discussion by MoroseWolf** [SevereCloud/vksdk#287] [Q&A]
Правильно понимаю что пока что библиотекой не предусмотрено отслеживание у объекта Video [статуса трансляции](https://dev.vk.com/ru/reference/objects/video#live_status)?
Делаю функцию оповещения о начавшейся трансляции у бота и как понимаю другой возможности это сделать нет.

**Comment by SevereCloud** [comment]
Да, создал ишью #288. Как вариант можно сейчас сделать свою обертку со своими структурами
